### DISCUSSION TITLE
Allowing bot to upload Music

### DISCUSSION THREAD

**Discussion by frostypirate** [Audionut/Upload-Assistant#86] [Ideas]
Hey there, I know this is a long shot but would it be possible to get this fork of L4G to allow us to upload Music using the bot? this would be awesome and help my site out millions if it can be done

**Comment by backstab5983** [comment]
Not sure what your site is, but this might come in handy: https://github.com/ghoto/smoked-salmon/tree/elghoto-version

**Comment by frostypirate** [comment]
my site is based on Unit3d source code
